config/sunspot.yml

```diff
@@ -1,24 +1,23 @@
----
 production:
   solr:
-    hostname: pad.publiclab.org
+    hostname: localhost
     port: 8983
-    read_timeout: 120
     log_level: WARNING
-    path: "/solr/default"
-  disabled: true
```

Comment:
Ah looks like maybe a problem while merging upstream. What I do is doing a rebase  instead of a merge from upstream. So I keep my master in sync with the upstream and pull into the master first, every time before rebasing. Rebasing keeps a cleaner commit history than merging.